expertino/clips/orders/order-selection.clp

```diff
@@ -62,19 +62,32 @@
   ?set-f <- (pddl-set-goals (instance ?instance) (state DONE))
   (pddl-manager (node ?node))
   (pddl-instance (name ?instance) (busy-with FALSE) (state LOADED))
+  (planning-filter (action-names $?an))
   (expertino-msgs-plan-temporal-client (server ?server&:(eq ?server (str-cat ?node "/temp_plan"))))
   (not (planned-for-main))
   =>
   (printout green "Start planning" crlf)
   (bind ?goal (expertino-msgs-plan-temporal-goal-create))
   (assert (pddl-planner-call (context test-plan) (goal ?goal)))
   (expertino-msgs-plan-temporal-goal-set-field ?goal "pddl_instance" ?instance)
+  (expertino-msgs-plan-temporal-goal-set-field ?goal "action_names" ?an)
   (expertino-msgs-plan-temporal-send-goal ?goal ?server)
   (assert (planned-for-main))
 )
 
-(defrule action-apply-effect-test
-  (pddl-action (instance ?instance) (name carrier-to-input) (params grey1 grey2 cap-grey cs1 cs1-input))
+(defrule action-apply-effect-test-main-action
+  (pddl-action (instance rcll) (id ?action-id) (name bs-dispense) (params o2-gen1 bs ?bs-side base-black ring-blue1))
   =>
-  (assert (pddl-action-apply-effect (instance ?instance) (name carrier-to-input) (params grey1 grey2 cap-grey cs1 cs1-input) (effect-type ALL)))
+  (assert (pddl-action-apply-effect (instance rcll) (action ?action-id) (effect-type ALL)))
+)
+
+(defrule action-apply-effect-test-sub-action
+  (pddl-action (instance rcll) (id ?dispense-id) (name bs-dispense))
```

Comment:
oh these are just tests? then ignore previous comment. Otherwise, why is this fixed for bs-dispense?

Reply:
got it!